packages/theming/theme-tokens/src/dark-macos/reactnative/tokens-aliases.json

```diff
@@ -1,262 +1,262 @@
-	}
-}
\ No newline at end of file
+  "body": {
+    "fontFamily": "\"Helvetica Neue\", Helvetica, Arial, \"Microsoft Sans Serif\"",
+    "fontLineHeight": 20,
+    "fontSize": 14,
+    "fontWeight": "400"
+  },
+  "bodyStrong": {
+    "fontFamily": "\"Helvetica Neue\", Helvetica, Arial, \"Microsoft Sans Serif\"",
+    "fontLineHeight": 20,
+    "fontSize": 14,
+    "fontWeight": "600"
+  },
+  "borderless": {
+    "strokeAlignment": "inner",
+    "strokeColorRest": "transparent",
+    "strokeColorHover": "transparent",
+    "strokeColorPressed": "transparent",
+    "strokeColorDisabled": "transparent",
+    "strokeWidth": 1
+  },
+  "brandBackground": {
+    "fillColorRest": "#185abd",
+    "fillColorHover": "#1651aa",
+    "fillColorPressed": "#0e336a",
+    "fillColorSelected": "#13458f"
+  },
+  "brandBackground2": {
+    "fillColorRest": "#d2e0f4"
+  },
+  "brandBackgroundStatic": {
+    "fillColorRest": "#185abd"
+  },
+  "brandForeground1": {
+    "fillColorRest": "#aec6eb"
+  },
+  "brandForeground2": {
+    "fillColorRest": "#d2e0f4"
+  },
+  "brandForegroundLink": {
+    "fillColorRest": "#aec6eb",
+    "fillColorHover": "#d2e0f4",
+    "fillColorPressed": "#2e6ac5",
+    "fillColorSelected": "#d2e0f4"
+  },
+  "brandStroke1": {
+    "strokeColorRest": "#185abd",
+    "strokeWidth": 1
+  },
+  "brandStroke2": {
+    "strokeColorRest": "#6794d7",
+    "strokeWidth": 1
+  },
+  "caption": {
+    "fontFamily": "\"Helvetica Neue\", Helvetica, Arial, \"Microsoft Sans Serif\"",
+    "fontLineHeight": 16,
+    "fontSize": 12,
+    "fontWeight": "400"
+  },
+  "compoundBrandBackground1": {
+    "fillColorRest": "#185abd",
+    "fillColorHover": "#1651aa",
+    "fillColorPressed": "#0e336a"
+  },
+  "compoundBrandForeground1": {
+    "fillColorRest": "#aec6eb",
+    "fillColorHover": "#d2e0f4",
+    "fillColorPressed": "#2e6ac5"
+  },
+  "compoundBrandStroke1": {
+    "strokeColorRest": "#185abd",
+    "strokeColorHover": "#1651aa",
+    "strokeColorPressed": "#0e336a",
+    "strokeWidth": 1
+  },
+  "display": {
+    "fontFamily": "\"Helvetica Neue\", Helvetica, Arial, \"Microsoft Sans Serif\"",
+    "fontLineHeight": 92,
+    "fontSize": 68,
+    "fontWeight": "600"
+  },
+  "headline": {
+    "fontFamily": "\"Helvetica Neue\", Helvetica, Arial, \"Microsoft Sans Serif\"",
+    "fontLineHeight": 28,
+    "fontSize": 20,
+    "fontWeight": "600"
+  },
+  "largeTitle": {
+    "fontFamily": "\"Helvetica Neue\", Helvetica, Arial, \"Microsoft Sans Serif\"",
+    "fontLineHeight": 52,
+    "fontSize": 40,
+    "fontWeight": "600"
+  },
+  "neutralBackground1": {
+    "fillColorRest": "#292929",
+    "fillColorHover": "#3d3d3d",
+    "fillColorPressed": "#1f1f1f",
+    "fillColorSelected": "#383838"
+  },
+  "neutralBackground2": {
+    "fillColorRest": "#1f1f1f",
+    "fillColorHover": "#333333",
+    "fillColorPressed": "#141414",
+    "fillColorSelected": "#2e2e2e"
+  },
+  "neutralBackground3": {
+    "fillColorRest": "#555555",
```

Comment:
This difference is still here

Can the changes to these files be fully reverted? Or can we update the input files instead if these entries are supposed to change? This file is auto-generated, we should not be editing it directly (same with the light version of this file)

Reply:
(same comment for neutralForeground3)

Reply:
I did `git checkout origin/master <tokens-aliases.json>`, that should revert the file to its original state.